### PR TITLE
3254 upcoming trial session

### PR DESCRIPTION
**Pull request by matthopson**

### CODE REVIEW

File: web-client/src/views/CaseDetail/AddToTrialModal.jsx [hunk 1]
```diff
@@ -127,6 +128,13 @@ export const AddToTrialModal = connect(
                 )}
             </BindedSelect>
           </FormGroup>
+          {addToTrialSessionModalHelper.showSessionNotSetAlert && (
+            <Hint>
+              This trial session has not been set. This case will be added to
+              the eligilble cases for this session and prioritized when the
```
Comment (rachaelparris) [inline web-client/src/views/CaseDetail/AddToTrialModal.jsx]: Typo here - `eligilble`

Comment (matthopson) [reply web-client/src/views/CaseDetail/AddToTrialModal.jsx]: Copy/pasted from Figma :P haha

File: shared/src/business/useCases/trialSessions/getEligibleCasesForTrialSessionInteractor.js [hunk 2]
```diff
@@ -32,17 +32,32 @@ exports.getEligibleCasesForTrialSessionInteractor = async ({
       trialSessionId,
     });
 
+  let calendaredCases = [];
+  if (trialSession.isCalendared === false && trialSession.caseOrder) {
```
Comment (rachaelparris) [inline shared/src/business/useCases/trialSessions/getEligibleCasesForTrialSessionInteractor.js:38]: I think a comment here would be helpful to explain that the manually added cases are stored as "calendared" cases even though the session isn't calendared.

**Review by rachaelparris — APPROVED** [review]

File: shared/src/business/useCases/trialSessions/getEligibleCasesForTrialSessionInteractor.js (hunk 2 above)
Comment (matthopson) [reply shared/src/business/useCases/trialSessions/getEligibleCasesForTrialSessionInteractor.js:38]: Good call

**Pull request merged**